api/api.py

```diff
@@ -597,3 +597,19 @@ def enable_feature(feature_name: str) -> None:
   Args:
     feature_name: name of the feature to enable
   """
+  pass
+
+def local_resource(name: str, cmd: str, deps: Union[str, List[str]] = None, trigger_mode: TriggerMode = TRIGGER_MODE_AUTO) -> None:
+  """Configures `cmd` to run on the _host_ machine, not in a remote cluster.
+
+  If `deps` is set then `cmd` is only run if one of the files specified changes.
```

Comment:
```suggestion
  If `deps` is set then `cmd` is run whenever one of the files specified changes.
```
(consider something like this because technically that initial run of this command is not triggered by a change to one of these files)